application/account-management/WebApp/src/app/page.tsx

```diff
@@ -1,8 +1,12 @@
+import { Trans, t } from "@lingui/macro";
+import { LocaleSwitcher } from "@/ui/LocaleSwitcher";
+
 export default function Page() {
   return (
     <div>
-      <h2>Root</h2>
-      <p>This is the main page</p>
+      <h2>
+        <Trans>Overview</Trans>
```

Comment:
Is `<Trans>` our own html tag. Anyway, we can call it `<translation>` (I guess we also want it lowercase?

Reply:
It's from the lingui package - would personally prefer:
```
t`Overview`
```

We could create an alias but developers reading the library docs might find it confusing